### PR TITLE
Update Wither skeleton Drops

### PR DESCRIPTION
**Pull request by good777LUCKY**
Update Wither skeleton Drops

### CODE REVIEW

File: src/main/java/nukkitcoders/mobplugin/entities/monster/walking/WitherSkeleton.java [hunk 1]
```diff
@@ -79,17 +79,23 @@ public void spawnTo(Player player) {
...
+            drops.add(Item.get(Item.COAL, 0, 1));
         }
-
+        
+        if (Utils.rand(1, 40) == 1) {
+            drops.add(Item.get(Item.SKULL, 1, 1));
+        }
+        
+        if (Utils.rand(1, 200) <= 17) {
+            drops.add(Item.get(Item.STONE_SWORD, 0, 1));
```
Comment (PetteriM1) [inline src/main/java/nukkitcoders/mobplugin/entities/monster/walking/WitherSkeleton.java:96]: I think the sword should have a random durability

Comment (good777LUCKY) [reply src/main/java/nukkitcoders/mobplugin/entities/monster/walking/WitherSkeleton.java]: ok

File: src/main/java/nukkitcoders/mobplugin/entities/monster/walking/WitherSkeleton.java [hunk 2]
```diff
@@ -79,17 +79,23 @@ public void spawnTo(Player player) {
...
+            drops.add(Item.get(Item.COAL, 0, 1));
         }
-
+        
+        if (Utils.rand(1, 40) == 1) {
+            drops.add(Item.get(Item.SKULL, 1, 1));
+        }
+        
+        if (Utils.rand(1, 200) <= 17) {
+            drops.add(Item.get(Item.STONE_SWORD, 0, 1).setDamage(Utils.rand(0, 131)));
```
Comment (PetteriM1) [inline src/main/java/nukkitcoders/mobplugin/entities/monster/walking/WitherSkeleton.java:96]: Damage is the same as the meta value so you don't have to use `setDamage` separately here

Comment (good777LUCKY) [reply src/main/java/nukkitcoders/mobplugin/entities/monster/walking/WitherSkeleton.java]: Oh, i was confused

**Pull request merged**